pkg/topkg.ml

```diff
@@ -27,8 +27,12 @@ module type Exts = sig
   val interface_opt : string list
   (** [interface_opt] is [".cmx" :: interface] *)
 
+  val platform_lib_extension : string
+  (** Extension of the native platform library, ".a" for
```

Comment:
\o/ a Windows user ! 

`Exts.platform_lib_extension` is a very long winded and redundant name. Besides, values from this module are lists of extensions I would suggest: 
```
val c_library : string list 
```
Other than that I'm sure similar business needs to be done for dlls (e.g. [here](https://github.com/dbuenzli/mtime/blob/master/pkg/build.ml#L23)) could you update the PR with a right: 
```
val c_dll_library : string list
```